framework/include/wifi_manager/wifi_manager.h

```diff
@@ -127,6 +136,19 @@ typedef struct {
 	char passphrase[64];
 } wifi_manager_softap_config_s;
 
+/**
+ * @brief Specify the policy of reconnect when the device is disconnected
+ */
+typedef struct {
+	wifi_manager_reconn_type_e type;
+	// interval: if type is INTERVAL, it will try to connect AP every interval second
+	//           if type is EXPONENTIAL, it is initial wait time.
+	int interval;
+	// max_interval: it is the maximum wait time if type is EXPONENTIAL
+	//             : it is not used if type is INTERVAL
+	int max_interval; // 
+} wifi_manager_connect_config_s;
```

Comment:
Is this structure used only when reconnecting?
Then for the structure name, wifi_manager_reconnect_config_s might be better than wifi_manager_connect_config_s.

Reply:
ok I will change